src/main/java/com/revature/byteshare/favorites/favoriteService.java

```diff
@@ -0,0 +1,59 @@
+package com.revature.byteshare.favorites;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class favoriteService {
```

Comment:
Class should be Pascal Case, like FavoriteService, follow this format for all files